resources/assets/js/components/UpcomingColumn.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
```

Comment:
Can we put this file in a new folder called Upcoming?

Reply:
No.

Reply:
Ok! Thanks for your suggestion. I am working on it right now.

Reply:
Done!